File: src/app/book-detailed-dto.ts
export class BookDetailedDTO {


    bookId:number;
    title:string;
    author:string;
    language:string;
    category:string;
    price:number;
    discount:number;
    publishDate:string;
    imageName:string;
    description:string;

    constructor(
    bookId:number,
    title:string,
    author:string,
    language:string,
    category:string,
    price:number,
    discount:number,
    publishDate:string,
    imageName:string,description:string

    )
    {
        this.bookId=bookId;
        this.title=title;
        this.author=author;
        this.language=language;
        this.category=category;
        this.price=price;
        this.discount=discount;
        this.publishDate=publishDate;
        this.imageName=imageName;
        this.description=description;
    }
}

File: src/main/java/com/cg/bookstoreapplication/dto/BookDetailedDTO.java
package com.cg.bookstoreapplication.dto;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BookDetailedDTO {

	private int bookId;
	private String title;
	private String author;
	private String category;
	private String description;
	private int discount;
	private double price;
	private String publishDate;
	private String language;
	private String imageName;
	
}
